pai-management/k8sPaiLibrary/template/kubelet.sh.template

```diff
@@ -19,13 +19,18 @@
 
 set -e
 
+
+DOCKER_ROOT_DIR_FOR_KUBELET=`docker info | grep "Docker Root Dir" | awk -F' ' '{print $NF}'`
```

Comment:
Hi @ydye ,
Could we using Docker client api for this purpose?

Reply:
What's the benefit?

Reply:
API is more readable and compatible.
Shell script heavily dependent on the command line output format, it's hard to assume the result without execute it.

Reply:
But we will have to install docker-client into the remote host.

Reply:
I see.
In case that docker client could connect remote daemon, we may not need to execute script on remote host.

Reply:
This will introduce another dependency, and make deployment more complicated

Reply:
The command `docker info` on my VM or in the dev-box always show a warning `no swap limit support`, so I think use python api is better:
![image](https://user-images.githubusercontent.com/11583225/45008844-6a6cb880-b037-11e8-80a3-6d986d202fd4.png)

Reply:
This message is okay.